src/components/ConversationSettings/Matterbridge/matterbridgeTypes.ts

```diff
@@ -0,0 +1,386 @@
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+import { t } from '@nextcloud/l10n'
+import { imagePath } from '@nextcloud/router'
+
+type InputField = {
+	type: 'url' | 'text' | 'password',
+	placeholder: string,
+	icon: string,
+} | {
+	type: 'checkbox',
+	labelText: string,
+}
+type MatterbridgeType = {
+	name: string,
+	iconUrl: string,
+	infoTarget: string,
+	fields: Record<string, InputField>
+	mainField: string,
+}
+
+export const matterbridgeTypes: Record<string, MatterbridgeType> = {
+	nctalk: {
+		name: 'Nextcloud Talk',
+		iconUrl: imagePath('spreed', 'app-dark.svg'),
+		infoTarget: 'https://github.com/42wim/matterbridge/wiki/Section-Nextcloud-Talk-%28basic%29',
+		fields: {
+			server: {
+				type: 'url',
+				placeholder: t('spreed', 'Nextcloud URL'),
+				icon: 'icon-link',
+			},
+			login: {
+				type: 'text',
+				placeholder: t('spreed', 'Nextcloud user'),
+				icon: 'icon-user',
+			},
+			password: {
+				type: 'password',
+				placeholder: t('spreed', 'User password'),
+				icon: 'icon-category-auth',
+			},
+			channel: {
+				type: 'text',
+				placeholder: t('spreed', 'Talk conversation'),
```

Comment:
Change to `Talk conversation token` ?

Reply:
Follow-up, BridgePart needs migration to Vue components, and it's unrelated to fix TBH